site/content/docs/user/using-wsl2.md

```diff
@@ -66,6 +66,48 @@ Once your Windows Insider machine is ready, you need to do a few more steps to s
 
 Now, move on to the [Quick Start](/docs/user/quick-start) to set up your cluster with kind.
 
+## Troubleshooting
+
+### Failures with `kind cluster create`
```

Comment:
this renders a bit weird to me, is this on purpose using the code marks on the title?

Reply:
https://deploy-preview-984--k8s-kind.netlify.com/docs/user/using-wsl2/#troubleshooting

Reply:
:man_facepalming: 
I always forget I can see the preview on that link
sorry